{"name": "test_firstTest", "status": "failed", "statusDetails": {"message": "AssertionError: assert False", "trace": "self = <test_myFirstTest.TestMush object at 0x1037ab2f0>\n\n    def test_firstTest(self):\n        print('First test1')\n        print('Second test')\n>       assert False\nE       assert False\n\nsrc/test_myFirstTest.py:5: AssertionError"}, "start": 1745219439158, "stop": 1745219439158, "uuid": "5013a4f7-b4f7-4561-96c9-f2a71555fe6c", "historyId": "98b9d2a354c1b40711ae7fd21ed656bb", "testCaseId": "98b9d2a354c1b40711ae7fd21ed656bb", "fullName": "src.test_myFirstTest.TestMush#test_firstTest", "labels": [{"name": "parentSuite", "value": "src"}, {"name": "suite", "value": "test_myFirstTest"}, {"name": "subSuite", "value": "TestMush"}, {"name": "host", "value": "Musheer-Khan.local"}, {"name": "thread", "value": "22481-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "src.test_myFirstTest"}]}